addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2485,7 +2485,7 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
         }
         options.domain = JSON.parse(options.domain);
         options.nullText = $target[0].dataset['nullText'] ||
-            JSON.parse($target[0].dataset['oeContactOptions'])['null_text'];
+            JSON.parse($target[0].dataset['oeContactOptions'] || '{}')['null_text'];
```

Comment:
It would be nicer to move each of the fix of this commit to their origin commit - and ultimately remove this commit.

Reply:
Do you mean to split the commit in as many commits as fixes it includes?

Reply:
Yes, then put each one inside one of the previous commits.

Reply:
Done!

Maybe you would like to add a more detailed description to the `fix handling of possibly missing json` commit. 😉

Reply:
No need: it should just be "fixup" within the initial commit where that line appeared.

Reply:
Alright, I'm squashing it 👍